osmnx/utils.py

```diff
@@ -573,6 +618,125 @@ def get_nearest_nodes(G, X, Y, method=None):
     return np.array(nn)
 
 
+def get_nearest_edges(G, X, Y, method=None, dist=10):
```

Comment:
The default *dist* should be changed to degrees. Any suggestions for a default value?